python/cudf/cudf/tests/test_stats.py

```diff
@@ -316,6 +316,37 @@ def test_series_median(dtype, num_na):
         np.testing.assert_approx_equal(actual, desired)
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        np.random.normal(-100, 100, 1000),
+        np.random.randint(-50, 50, 1000),
+        np.zeros(100),
+        np.array([1.123, 2.343, np.nan, 0.0]),
+        np.array([-2, 3.75, 6, None, None, None, -8.5, None, 4.2]),
+        cudf.Series([]),
+        cudf.Series([-3]),
+    ],
+)
+@pytest.mark.parametrize("periods", range(-5, 5))
+@pytest.mark.parametrize("fill_method", ["ffill", "bfill"])
+def test_series_pct_change(data, periods, fill_method):
+    cs = cudf.Series(data)
+    ps = cs.to_pandas()
+
+    if np.abs(periods) <= len(cs):
```

Comment:
This means the last two test cases `cudf.Series([])` and `cudf.Series([-3])` is untested.

Reply:
The tested periods range from -5 to 5. The first will be tested when `periods` is equal to 0 and the second will be tested when `periods` is either -1, 0, or 1.

Reply:
Oops. Yup that's right.